src/status_im2/contexts/wallet/account/tabs/about/view.cljs

```diff
@@ -53,17 +53,32 @@
 
 (defn- view-internal
   []
-  [rn/view {:style style/about-tab}
-   [quo/data-item
-    (merge temp/data-item-state
-           {:custom-subtitle (fn [] [quo/address-text
-                                     {:networks [{:network-name :ethereum :short-name "eth"}
-                                                 {:network-name :optimism :short-name "opt"}
-                                                 {:network-name :arbitrum :short-name "arb1"}]
-                                      :address  temp/address
-                                      :format   :long}])
-            :container-style {:margin-bottom 12}
-            :on-press        #(rf/dispatch [:show-bottom-sheet {:content about-options}])})]
-   [quo/account-origin temp/account-origin-state]])
+  (let [profile                (rf/sub [:profile/profile-with-image])
+        {:keys [address path]} (rf/sub [:wallet/current-viewing-account])]
+    [rn/view {:style style/about-tab}
+     [quo/data-item
+      {:description     :default
+       :icon-right?     true
+       :right-icon      :i/options
+       :card?           true
+       :label           :none
+       :status          :default
+       :size            :default
+       :title           (i18n/label :t/address)
+       :custom-subtitle (fn [] [quo/address-text
+                                {:networks [{:network-name :ethereum :short-name "eth"}
+                                            {:network-name :optimism :short-name "opt"}
+                                            {:network-name :arbitrum :short-name "arb1"}]
+                                 :address  address
+                                 :format   :long}])
+       :container-style {:margin-bottom 12}
+       :on-press        #(rf/dispatch [:show-bottom-sheet {:content about-options}])}]
+     [quo/account-origin
+      {:type            :default-keypair
+       :stored          :on-device
+       :profile-picture (profile.utils/photo profile)
+       :derivation-path path
+       :user-name       (profile.utils/displayed-name profile)
+       :on-press        #(js/alert "pressed")}]]))
```

Comment:
nit, I think we should say "to be implemented" instead. or something to that effect.